eslint/eslint-patch/src/eslint-bulk-suppressions/ast-guards.ts

```diff
@@ -209,9 +209,9 @@ export function isNormalAssignmentPatternWithAnonymousExpressionAssigned(
   return isNormalAssignmentPattern(node) && isNormalAnonymousExpression(node.right);
 }
 
-export interface INormalVariableDeclaratorWithAnonymousExpressionAssigned extends INormalVariableDeclarator {
+export type INormalVariableDeclaratorWithAnonymousExpressionAssigned = INormalVariableDeclarator & {
```

Comment:
Curious why the change? Is this not functionally identical?

Reply:
`INormalVariableDeclarator` was changed to be a type union, and you can't have something extend that.